doc/source.md

```diff
@@ -0,0 +1,221 @@
+
+```json
+ {
+  "name": "TDengineSourceConnector",
+  "config": {
+    "connector.class": "com.taosdata.kafka.connect.source.TDengineSourceConnector",
+    "tasks.max": 1,
+    "connection.url": "jdbc:TAOS://127.0.0.1:6030",
+    "connection.username": "root",
+    "connection.password": "taosdata",
+    "connection.database": "source",
+    "connection.attempts": 3,
+    "connection.backoff.ms": 1000,
+    "topic.prefix": "tdengine-",
+    "poll.interval.ms": 1000,
+    "fetch.max.rows": 100,
+    "out.format": "line",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.storage.StringConverter"
+  }
+}
+```
+
+Run the connector with this configuration
+
+```shell
+curl -X POST -d @tdengine-source-connector.json http://localhost:8083/connectors -H "Content-Type:application/json"
+```
+
+## Configuration
+
+### connector.class
+
+To use this connector, specify the name of the connector class in the `connector.class` configuration property.
+
+```shell
+connector.class=com.taosdata.kafka.connect.source.TDengineSourceConnector
+```
+
+- Type:string
+- Importance:high
+- Default:null
+
+### tasks.max
+
+The connector can be configured to use as few as one task (tasks.max=1) or scale to as many tasks as required to capture all table changes.
+
+- Type:int
+- Importance:high
+- Default:1
+
+### connection.url
+
+The URL of the TDengine database to write to.
+
+- Type:string
+- Importance:high
+- Default:null
+
+### connection.username
+
+The username to connect to TDengine with. default value is `root`.
+
+- Type:string
+- Importance:high
+- Default:null
+
+### connection.password
+
+The password to connect to TDengine with. default value is `taosdata`.
+
+- Type:string
+- Importance:high
+- Default:null
+
+### connection.database
+
+The TDengine database name from which records have to be read and publish data to configured Apache Kafka® topic.
+
+- Type:string
+- Importance:high
+- Default:null
+
+### connection.attempts
+
+The maximum number of times to retry on errors before failing the connection.
+
+- Type:int
+- Importance:high
+- Default:3
+
+### connection.backoff.ms
+
+Backoff time duration to wait before retrying connection (in milliseconds).
+
+- Type:int
+- Importance:high
+- Default:5000
+
+### topic.prefix
+
+Prefix that should be prepended to super table names to generate the name of Apache Kafka® topic to publish to.
+
+- Type:string
+- Importance:high
+- Default: ""
+
+### timestamp.initial
+
+The timestamp used for initial queries. If not specified, all data will be retrieved. format is `yyyy-MM-dd HH:mm:ss` 
+
+- Type:string
```

Comment:
need space after ":"
you may need a markdown format tool.

Reply:
do as you say